packages/swing-store/src/transcriptStore.js

```diff
@@ -281,13 +370,10 @@ export function makeTranscriptStore(db, ensureTxn, noteExport) {
     if (startPos === undefined) {
       ({ startPos, endPos } = getCurrentSpanBounds(vatID));
     } else {
-      assert.typeof(startPos, 'number');
+      insistTranscriptPosition(startPos);
       endPos = sqlGetSpanEndPos.get(vatID, startPos);
-      assert.typeof(
-        endPos,
-        'number',
-        `no transcript span for ${vatID} at ${startPos}`,
-      );
+      typeof endPos === 'number' ||
+        Fail`no transcript span for ${q(vatID)} at ${q(startPos)}`;
     }
     insistTranscriptPosition(startPos);
```

Comment:
Is this now unneeded? Or just there to make TS happy?

Reply:
More suspenders-and-belt, making sure internal invariants are.

Reply:
I'm just surprised we're checking this twice in the case of `startPos === undefined`.